### ISSUE TITLE
Implement new functionality for the admin chat

### ISSUE THREAD

**Issue by Boyan-Apostolov** [Berat-Dzhevdetov/DeemZ-Platform#34]
Let's keep the admin chat message history in the DB.
Also, we can implement sending images after pasting them from the clipboard. We can use [this JavaScript code](https://pastebin.com/hbrnLpex) for that. Any other ideas?

**Comment by Berat-Dzhevdetov** [comment]
We should do something when someone pastes a non-image file in the input because currently, it's breaking the chat. Also, we should give name attribute to the images so we can send them through the form and maybe it's a good idea to implement drag & drop funcitonality.

**Comment by Boyan-Apostolov** [comment]
I improved showing the attached images, can you make the input textarea look better?

**Comment by Boyan-Apostolov** [comment]
We can save the images' links in the DB after we decode them with [this code](https://pastebin.com/7yw4wZm4).

**Comment by Berat-Dzhevdetov** [comment]
I don't think we should save the image in the database. This could affect the productivity of the platform.

**Comment by Boyan-Apostolov** [comment]
OK Mr, Where shall we save the image then?

**Comment by Berat-Dzhevdetov** [comment]
That's why we have the cloud. The only things we should save in the database is the public key and the URL from the cloud.
